### DISCUSSION TITLE
How to get records based on foreign (embedded) table's column value?

### DISCUSSION THREAD

**Discussion by heauton** [supabase/supabase#1827] [Questions] [answered]
I am using the JavaScript library and couldn't figure out how to filter items based on the foreign table's column value.

Say you want to fetch a list of comments and but only if the post is not `status: deleted` or `is_deleted` - very contrived example but basically retrieving child records based on some value from a parent record. 

I tried some filters documented in https://supabase.io/docs/reference/javascript/using-filters nothing rings a bell. I got the embedded record as null (so using the above example `post: null`) but I still got the list of comments left on this deleted post.

Any clues?

**Comment by heauton** [comment]
Oh, forgot to mention that I could use RLS to achieve the expected result but I am exploring ways to do this in the browser.

**Comment by soedirgo** [comment, marked as answer]
Hey @heauton, this isn't possible right now, you can filter on foreign table columns but they don't affect the results from the original table.

**Reply by heauton** [reply]
Thanks @soedirgo whenever I get stuck I look up posts here in discussions and all your responses have been really helpful :)

**Reply by irfaan** [reply]
@soedirgo will you notify us here if this does become a feature?

**Reply by soedirgo** [reply]
@irfaan sure. This is currently being tracked here: https://github.com/supabase/postgrest-js/issues/197
